lib/langchain/llm/response/openai.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Langchain::LLM::Response
+  class OpenAI < Base
+    attr_reader :raw_response
+
+    def model
+      raw_response["model"]
+    end
+
+    def type
+      return options[:type] if options[:type]
+
+      is_embedding? ? "embedding" : raw_response.dig("object")
+    end
+
+    def completions
+      raw_response.dig("choices")&.map { |choice| choice.dig("message") }
+    end
+
+    def embeddings
+      raw_response.dig("data")&.map { |datum| datum.dig("embedding") }
+    end
+
+    def value
+      is_embedding? ? embeddings.first : completions.first.dig("content")
+    end
+
+    def prompt_tokens
+      raw_response.dig("usage", "prompt_tokens")
+    end
+
+    def completion_tokens
+      raw_response.dig("usage", "completion_tokens")
+    end
+
+    def total_tokens
```

Comment:
Thoughts on calling these: `total_tokens_used`, `completion_tokens_used`, `prompt_tokens_used`? So it's even more clear.

Reply:
seems like just more characters to type to me 😄

Reply:
I was wondering if it brings more clarity?

Reply:
I'm not sure if `used` is bringing more clarity.. It is usage from OpenAI API limits perspective, but in context of Response object it is just counts of tokens in prompt and completion. 

I'd keep it short and named according to the class context, rather than overall paradigm of API's tokens usage/limits.